### PR TITLE
Homework 7 Armine Zakaryan

### PR DESCRIPTION
**Pull request by ArmineZak**

### CODE REVIEW

File: src/annotations/Main.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package annotations;
+
+import java.time.LocalDate;
+
+public class Main {
+    public static void main(String[] args) throws IllegalAccessException {
+        LocalDate birthdate = LocalDate.of(2009, 01, 01);
```
Comment (vahan1994vs) [inline src/annotations/Main.java:7]: LocalDate.of(2009, 1, 1);

File: src/annotations/Validator.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator<T> {
+
+    public List<?> validate(T object) throws IllegalAccessException {
+        List<String> list = new ArrayList();
```
Comment (vahan1994vs) [inline src/annotations/Validator.java:13]: List<String> list = new ArrayList<>();

File: src/annotations/Validator.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        Class objectClass = object.getClass();
```
Comment (vahan1994vs) [inline src/annotations/Validator.java:14]: Class<?> objectClass = object.getClass();

**Review by vahan1994vs — COMMENTED** [review]
reviewed

File: src/annotations/Validator.java (hunk 3 above)
Comment (ArmineZak) [reply src/annotations/Validator.java]: Corrected.

File: src/annotations/Validator.java (hunk 2 above)
Comment (ArmineZak) [reply src/annotations/Validator.java]: Corrected.

**Pull request closed without merging**